notebooks/official/workbench/fraud_detection/fraud-detection-model.ipynb

```diff
@@ -219,11 +219,12 @@
       "source": [
```

Comment:
<p><strong>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</strong></p><p> </p><p><strong>Combine the two cells</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/821/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>